great_expectations/profiler/util.py

```diff
@@ -14,61 +17,157 @@
 )
 
 
+def domain_kwargs_parameter(func: Callable = None) -> Callable:
+    @wraps(func)
```

Comment:
I would love to better understand the reasoning behind this decorator

Reply:
It has been removed.